File: Modul 6/Python/PRAK603-2310817310010-AkmallullailSyaban.py
n1, n2 = map(int, input().split())
hasil = []
if n1 != n2:
    print("Jumlah Tidak Sama")
else:
    scan1 = list(map(int, input().split()))
    scan2 = list(map(int, input().split()))

    for i in range(n1):
        a = scan1[i] * scan2[i]
        hasil.append(a)
        print(hasil[i], end=" ")

File: Modul 6/C/PRAK603-2310817310010-AkmallullailSyaban.c
#include <stdio.h>

int main() {
    int n1, n2;
    scanf("%d %d", &n1, &n2);
    int scan1[n1], scan2[n2];

    if (n1 != n2) {
        printf("Jumlah tidak sama");
    } else {
        for(int i = 0; i < n1; i++){
        scanf("%d", &scan1[i]);
        }
        for(int i = 0; i < n2; i++){
        scanf("%d", &scan2[i]);
        printf("%d ", scan1[i] * scan2[i]);
        }
    }
    return 0;
}